kafka-init/create_topics.py

```diff
@@ -177,10 +205,12 @@ def main():
     existing_topics = list_topics()
     logger.info('Kafka has topics: %r', existing_topics)
 
-    created_topics = create_topics(default_config, existing_topics)
+    created_topics, existing_topics_config = create_topics(default_config, existing_topics)
     logger.info('Topic creation finished successfully. Created: %r',
                 created_topics)
 
+    logger.info(existing_topics_config)
```

Comment:
This message should probably be more descriptive or removed

Reply:
Agree, but do we want to address that in a follow-up?

Reply:
i can change